README.md

```diff
@@ -55,7 +55,12 @@ By default, action is designed to run with minimal configuration but you can alt
-| SLACK_THREAD_TS          | -                                                     | If you want to send message in a thread, you can pass the timestamp of the parent message to this variable. You can get the timestamp of the parent message from the message URL in Slack. (ex: `SLACK_THREAD_TS: 1586130833.000100`)                                                                                    |
+| SLACK_THREAD_TS          | -                                                     | If you want to send message in a thread, you can pass the timestamp of the parent message to this variable. You can get the timestamp of the parent message from the message URL in Slack. (ex: `SLACK_THREAD_TS: 1586130833.000100`). To obtain the timestamp of the parent message, follow these steps:                   |
+|                          |                                                       | 1. Open the Slack message you want to reply to in a thread.                                                                                                                                                                                                                                                              |
+|                          |                                                       | 2. Click on the "More actions" button (three vertical dots) on the message.                                                                                                                                                                                                                                              |
+|                          |                                                       | 3. Select "Copy link" from the dropdown menu.                                                                                                                                                                                                                                                                            |
+|                          |                                                       | 4. Paste the copied link in a text editor or any other place where you can see the full URL.                                                                                                                                                                                                                             |
+|                          |                                                       | 5. The timestamp of the parent message is the last part of the URL, which looks like `1586130833.000100`.                                                                                                                                                                                                                 |
```

Comment:
Could we add a mention here that it's now possible to get a reference to a previously posted message via the `ts` output?